kernel/src/fs/configfs/fs.rs

```diff
@@ -38,7 +38,12 @@ impl ConfigFs {
     pub(super) fn singleton() -> &'static Arc<ConfigFs> {
         static SINGLETON: Once<Arc<ConfigFs>> = Once::new();
 
-        SINGLETON.call_once(|| Self::new(ConfigRootNode::singleton().clone()))
+        SINGLETON.call_once(|| Self::new(ConfigRootNode::singleton().clone()));
+
+        #[cfg(all(target_arch = "x86_64", feature = "cvm_guest"))]
+        crate::security::tsm::init();
```

Comment:
Do not call `tsm::init` directly here.

Check out the `kernel/lib.rs`. All top-level initialization functions are called there.

https://github.com/asterinas/asterinas/blob/9c70ac0f0a9f7411f5f136ce2f1b3db723f01ef0/kernel/src/lib.rs#L96-L105

So a more preferable approach is to append `security::init();` to the end of the function. And the `security::init` function will in turn call `tsm::init()`.

In addition, limit the visibility of `tsm::init` to `pub(super)` so that no one except the parent `security` module can call the initialization function of `tsm`.